server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -180,6 +184,43 @@ public Invocations getInvocations() {
       return completedFuture(responseBuilder.build());
    }
 
+   private CompletionStage<RestResponse> getCaches(RestRequest request) {
+      NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
+      if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
+
+      class CacheInfo {
```

Comment:
IMHO this class should be declared outside the method and also ```private static```

Reply:
allright